### PR TITLE
Jetpack Connect: Arrange Solution Cards Side-by-Side

### LINKED ISSUE #40679: The Scan & Backups cards are stacking on large screen (JP Connect page)
#### Steps to reproduce
On Desktop/laptop, Mac, chrome
1.WP admin Jetpack Connect prompt
2. Jetpack Connect Plans page

#### What I expected
I expected to see the backups and scan cards side by side

#### What happened instead
They are stacking 

#### Browser / OS version
Mac, chrome Version 80.0.3987.149
#### Screenshot / Video

Screenshot 2020-04-01 16 18 40

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

This ensures that the Solutions cards during Jetpack Connect break exactly like the Plans ones do.

#### Testing instructions

Follow the Jetpack Connect route to the Plans stage: `/jetpack/connect/plans/monthly/site`

At this point, verify that no issues occur with the columns, no matter how you resize the screen or how many there are. They should stack when the content is too small to fit in columns, but otherwise appear like this.

Screenshot 2020-04-10 at 12 06 32

cc @keoshi, @tyxla, @simison 

Fixes #40679

### CODE REVIEW

File: client/my-sites/plans-features-main/style.scss [hunk 1]
```diff
@@ -13,11 +13,20 @@
 			display: flex;
 			flex-flow: row wrap;
 
+			@include breakpoint( '>660px' ) {
+				flex-flow: row;
```
Comment (keoshi) [inline client/my-sites/plans-features-main/style.scss:17]:
```suggestion
				flex-flow: row;
				max-width: 1000px;
```

Comment (Aurorum) [reply client/my-sites/plans-features-main/style.scss:17]: Oops, that works much better - fixed, thank you! :)

**Comment by Aurorum** [comment]
Not sure how I didn't even notice that, oops, but that seems to work perfectly! Thank you, I've updated the PR (and feel free to push any changes if you'd like to!) :)

**Comment by Aurorum** [comment]
Thanks @keoshi! I've updated the description. :)

**Comment by keoshi** [comment]
@Aurorum Stellar! Thanks once again and stay safe.

**Comment by folletto** [comment]
Seems ok to me! 👍

**Comment by enejb** [comment]
Thanks for helping fix this @Aurorum!

**Pull request merged**